### ISSUE TITLE
Lost control over all lamps! :/

### ISSUE THREAD

**Issue by tnowak** [dresden-elektronik/deconz-rest-plugin#1813]
Not sure if this is right place but I have a major and urgent problem with Conbee II and/or deCONZ. I've bought Conbee II stick, paired some Philips, Osram and Aqara bulbs and first FLS-PP lp. All worked fine for about 2 weeks. Yesterday I had to reboot RPi with docker running deconz and I’ve totally lost control over all my lights! I can't even turn them off.......

API still works fine, my integration with FHEM and Loxone seem to change bulb icon state in deCONZ, but actual lights don't change their state. After couple of minutes deconz in started they all become greyed out!

![Clipboard01](https://user-images.githubusercontent.com/54801553/64109202-26fa7680-cd7f-11e9-8c77-6ce4bf01df57.jpg)

In Phoscon App I see such Conbee II details:
Version is 2.05.67 / 26.08.2019
Firmware 264A0700

In that Raspberry PI I have a z-stick also that occupies ACM0, so conbee is ACM1:
It is visible inside raspberry system direct:

root@loxberry:~# udevadm info /dev/ttyACM1
P: /devices/platform/soc/3f980000.usb/usb1/1-1/1-1.2/1-1.2:1.0/tty/ttyACM1
N: ttyACM1
S: serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE1996953-if00
S: serial/by-path/platform-3f980000.usb-usb-0:1.2:1.0
E: DEVLINKS=/dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE1996953-if00 /dev/serial/by-path/platform-3f980000.usb-usb-0:1.2:1.0
E: DEVNAME=/dev/ttyACM1
E: DEVPATH=/devices/platform/soc/3f980000.usb/usb1/1-1/1-1.2/1-1.2:1.0/tty/ttyACM1
E: ID_BUS=usb
E: ID_MODEL=ConBee_II
E: ID_MODEL_ENC=ConBee\x20II
E: ID_MODEL_ID=0030
E: ID_PATH=platform-3f980000.usb-usb-0:1.2:1.0
E: ID_PATH_TAG=platform-3f980000_usb-usb-0_1_2_1_0
E: ID_REVISION=0100
E: ID_SERIAL=dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE1996953
E: ID_SERIAL_SHORT=DE1996953
E: ID_TYPE=generic
E: ID_USB_CLASS_FROM_DATABASE=Communications
E: ID_USB_DRIVER=cdc_acm
E: ID_USB_INTERFACES=:020201:0a0000:
E: ID_USB_INTERFACE_NUM=00
E: ID_VENDOR=dresden_elektronik_ingenieurtechnik_GmbH
E: ID_VENDOR_ENC=dresden\x20elektronik\x20ingenieurtechnik\x20GmbH
E: ID_VENDOR_FROM_DATABASE=Dresden Elektronik
E: ID_VENDOR_ID=1cf1
E: MAJOR=166
E: MINOR=1
E: SUBSYSTEM=tty
E: TAGS=:systemd:
E: USEC_INITIALIZED=9876427694

I run deCONZ in docker with such variables:
DECONZ_WEB_PORT=8080
DECONZ_WS_PORT=8443
DECONZ_DEVICE=/dev/ttyACM1
mapped devices:
/dev/ttyACM1 --> /dev/ttyACM1
volume mapping:
/root/.local/share/dresden-elektronik/deCONZ --> /opt/deconz/
/etc/localtime --> /etc/localtime
network: host

Inside deconz docker ACM1 is also visible:
root@loxberry:/# udevadm info /dev/ttyACM1
P: /devices/platform/soc/3f980000.usb/usb1/1-1/1-1.2/1-1.2:1.0/tty/ttyACM1
N: ttyACM1
E: DEVNAME=/dev/ttyACM1
E: DEVPATH=/devices/platform/soc/3f980000.usb/usb1/1-1/1-1.2/1-1.2:1.0/tty/ttyACM1
E: MAJOR=166
E: MINOR=1
E: SUBSYSTEM=tty
and no other devices are available:
root@loxberry:/# udevadm info /dev/ttyACM0
Unknown device, --name=, --path=, or absolute path in /dev/ or /sys expected.
root@loxberry:/# udevadm info /dev/ttyAMA0
Unknown device, --name=, --path=, or absolute path in /dev/ or /sys expected.

This is log from deCONZ from docker start:

[marthoc/deconz] Starting deCONZ...,
[marthoc/deconz] Current deCONZ version: 2.05.66,
[marthoc/deconz] Web UI port: 8080,
[marthoc/deconz] Websockets port: 8443,
[marthoc/deconz] VNC Disabled,
libpng warning: iCCP: known incorrect sRGB profile,
This plugin does not support propagateSizeHints(),
This plugin does not support propagateSizeHints(),
This plugin does not support propagateSizeHints(),
12:20:51:122 HTTP Server listen on address 0.0.0.0, port: 8080, root: /usr/share/deCONZ/webapp/,
12:20:51:140 CTRL. 3.16.212:20:51:278 dev /dev/ttyAMA0,
12:20:51:278 COM: /dev/ttyACM1 / serialno: ,
12:20:51:278 COM: --dev: /dev/ttyACM1 (ConBee II),
12:20:51:278 ZCLDB init file /root/.local/share/dresden-elektronik/deCONZ/zcldb.txt,
12:20:51:594 parent process /bin/sh,
12:20:51:595 gw run mode: docker,
12:20:51:595 GW sd-card image version file does not exist: /root/.local/share/dresden-elektronik/deCONZ/gw-version,
12:20:51:595 sd-card cid: 035344535033324780ffffffff0135ab,
12:20:51:597 DB sqlite version 3.16.2,
12:20:51:598 DB PRAGMA page_count: 34,
12:20:51:598 DB PRAGMA page_size: 4096,
12:20:51:598 DB PRAGMA freelist_count: 1,
12:20:51:598 DB file size 139264 bytes, free pages 1,
12:20:51:598 DB PRAGMA user_version: 6,
12:20:51:598 DB cleanup,
12:20:51:602 DB create temporary views,
12:20:51:634 don't close database yet, keep open for 900 seconds,
12:20:51:635 started websocket server at port 8443,
12:20:51:642 discovery updated announce interval to 10 minutes,
12:20:51:648 found node plugin: libde_rest_plugin.so - REST API Plugin,
12:20:51:652 found node plugin: libde_signal_plugin.so - Signal Monitor Plugin,
12:20:51:677 found node plugin: libstd_otau_plugin.so - STD OTAU Plugin,
12:20:51:714 dev /dev/ttyAMA0,
12:20:51:714 COM: /dev/ttyACM1 / serialno: ,
12:20:51:714 COM: --dev: /dev/ttyACM1 (ConBee II),
12:20:51:802 Device firmware version 0x264A0700,
12:20:51:922 unlocked max nodes: 200,
12:20:52:012 Device protocol version: 0x010B,
12:20:52:024 new node - ext: 0x00212effff048730, nwk: 0x0000,
12:20:52:185 SensorNode 3 set node 0x00158d0001e88410,
12:20:52:198 SensorNode 2 set node 0x00158d0002b89976,
12:20:52:213 don't close database yet, keep open for 900 seconds,
12:20:52:213 LightNode 1: Aqara_Bulb_1 added,
12:20:52:227 don't close database yet, keep open for 900 seconds,
12:20:52:227 LightNode 2: Philips_Bulb_1 added,
12:20:52:239 SensorNode 5 set node 0x00158d0001e83532,
12:20:52:251 SensorNode 6 set node 0x00158d0002b898dc,
12:20:52:264 don't close database yet, keep open for 900 seconds,
12:20:52:264 LightNode 3: Aqara_Bulb_2 added,
12:20:52:276 SensorNode 7 set node 0x00158d0002e99acb,
12:20:52:277 SensorNode 8 set node 0x00158d0002e99acb,
12:20:52:288 SensorNode 9 set node 0x00158d00036bf606,
12:20:52:301 don't close database yet, keep open for 900 seconds,
12:20:52:301 LightNode 4: Osram_Bulb_1 added,
12:20:52:315 don't close database yet, keep open for 900 seconds,
12:20:52:316 LightNode 5: Osram_Bulb_2 added,
12:20:52:328 don't close database yet, keep open for 900 seconds,
12:20:52:329 LightNode 6: Osram_Bulb_3 added,
12:20:52:343 don't close database yet, keep open for 900 seconds,
12:20:52:343 LightNode 7: Osram_Bulb_4 added,
12:20:52:357 don't close database yet, keep open for 900 seconds,
12:20:52:358 LightNode 8: Osram_Bulb_5 added,
12:20:52:370 don't close database yet, keep open for 900 seconds,
12:20:52:371 LightNode 11: LED_Strip_1 added,
12:20:52:372 don't close database yet, keep open for 900 seconds,
12:20:52:372 LightNode 12: Light 12 added,
12:20:52:386 don't close database yet, keep open for 900 seconds,
12:20:52:386 LightNode 13: Philips_Spot_1 added,
12:20:52:398 don't close database yet, keep open for 900 seconds,
12:20:52:399 LightNode 14: Philips_Spot_2 added,
12:20:52:413 don't close database yet, keep open for 900 seconds,
12:20:52:413 LightNode 15: Osram_Bulb_6 added,
12:20:52:427 don't close database yet, keep open for 900 seconds,
12:20:52:427 LightNode 16: Osram_Bulb_7 added,
12:20:52:531 Current channel 11,
12:20:52:549 CTRL ANT_CTRL 0x03,
12:20:52:576 Device protocol version: 0x010B,
12:20:52:630 Current channel 11,
12:20:52:648 CTRL ANT_CTRL 0x03,
12:20:56:465 dev /dev/ttyAMA0,
12:20:56:466 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26490700.bin.GCF,
12:20:56:466 GW firmware version: 0x264a0700,
12:20:56:466 GW firmware version is up to date: 0x264a0700,
12:20:56:759 Announced to internet,
12:20:56:759 discovery server date: Mon, 02 Sep 2019 10:20:56 GMT,
12:20:56:759 	 local time seems to be ok,
12:20:56:759 discovery found version 2.04.35 for update channel stable,
12:20:57:981 New websocket 192.168.103.101:51868 (state: 3) ,
12:21:10:208 hue: 181, sat: 254,
12:21:10:214 x: 0.639283, y: 0.330599,
12:21:10:215 drop request to zombie (rx = 0),
12:21:10:217 drop request to zombie (rx = 0),
12:21:10:520 hue: 181, sat: 254,
12:21:10:520 x: 0.639283, y: 0.330599,
12:21:10:520 drop request to zombie (rx = 0),
12:21:10:610 drop request to zombie (rx = 0),
12:21:11:259 drop request to zombie (rx = 0),
12:21:11:569 drop request to zombie (rx = 0),
12:21:12:481 hue: 181, sat: 254,
12:21:12:481 x: 0.639283, y: 0.330599,
12:21:12:482 drop request to zombie (rx = 0),
12:21:12:511 drop request to zombie (rx = 0),
12:21:12:791 hue: 181, sat: 254,
12:21:12:791 x: 0.639283, y: 0.330599,
12:21:12:792 drop request to zombie (rx = 0),
12:21:12:811 drop request to zombie (rx = 0),
12:21:13:191 drop request to zombie (rx = 0),
12:21:13:501 drop request to zombie (rx = 0),
12:21:14:243 hue: 181, sat: 254,
12:21:14:243 x: 0.639283, y: 0.330599,
12:21:14:244 drop request to zombie (rx = 0),
12:21:14:311 drop request to zombie (rx = 0),
12:21:14:564 hue: 181, sat: 254,
12:21:14:564 x: 0.639283, y: 0.330599,
12:21:14:565 drop request to zombie (rx = 0),
12:21:14:611 drop request to zombie (rx = 0),
12:21:14:951 drop request to zombie (rx = 0),
12:21:15:268 drop request to zombie (rx = 0),
12:21:16:352 hue: 181, sat: 254,
12:21:16:353 x: 0.639283, y: 0.330599,
12:21:16:353 drop request to zombie (rx = 0),
12:21:16:411 drop request to zombie (rx = 0),
12:21:16:668 hue: 181, sat: 254,
12:21:16:668 x: 0.639283, y: 0.330599,
12:21:16:668 drop request to zombie (rx = 0),
12:21:16:711 drop request to zombie (rx = 0),
12:21:17:409 hue: 96, sat: 254,
12:21:17:409 x: 0.639661, y: 0.330299,
12:21:17:409 drop request to zombie (rx = 0),
12:21:17:410 drop request to zombie (rx = 0),
21:17:612 scan finished,
12:21:17:719 hue: 96, sat: 254,
12:21:17:719 x: 0.639661, y: 0.330299,
12:21:17:719 drop request to zombie (rx = 0),
12:21:17:811 drop request to zombie (rx = 0),
12:21:18:112 drop request to zombie (rx = 0),
12:21:18:422 drop request to zombie (rx = 0),
12:21:37:090 New websocket 127.0.0.1:54266 (state: 3) ,
12:21:48:943 0x00178801021D40B2 error APSDE-DATA.confirm: 0xD0 on task,
12:21:51:219 Current channel 11,
12:21:51:223 Device TTL 2138 s flags: 0x7,
12:22:00:740 0x84182600000E266D error APSDE-DATA.confirm: 0xD0 on task,
12:22:11:543 0x84182600000E279F error APSDE-DATA.confirm: 0xD0 on task,
12:22:22:442 0xF0D1B8000010741A error APSDE-DATA.confirm: 0xD0 on task,
12:22:33:343 0xF0D1B80000106F3D error APSDE-DATA.confirm: 0xD0 on task,
12:22:44:142 0xF0D1B800000FF2FC error APSDE-DATA.confirm: 0xD0 on task,
12:22:51:230 Current channel 11,
12:22:51:240 Device TTL 2078 s flags: 0x7,
12:22:54:944 0x00212EFFFF032B30 error APSDE-DATA.confirm: 0xD0 on task,
12:23:05:846 0x00212EFFFF032B30 error APSDE-DATA.confirm: 0xD0 on task,
12:23:16:644 0x0017880106BAE90B error APSDE-DATA.confirm: 0xD0 on task,
12:23:27:448 0x0017880106BAE729 error APSDE-DATA.confirm: 0xD0 on task,
12:23:38:355 0xF0D1B800000FFAF3 error APSDE-DATA.confirm: 0xD0 on task,
12:23:49:147 0xF0D1B800000FF301 error APSDE-DATA.confirm: 0xD0 on task,
12:23:51:230 Current channel 11,
12:23:51:239 Device TTL 2018 s flags: 0x7,
12:24:00:939 0x00178801021D40B2 error APSDE-DATA.confirm: 0xD0 on task,
12:24:12:745 0x84182600000E266D error APSDE-DATA.confirm: 0xD0 on task,
12:24:23:545 0x84182600000E279F error APSDE-DATA.confirm: 0xD0 on task,
12:24:34:444 0xF0D1B8000010741A error APSDE-DATA.confirm: 0xD0 on task,
12:24:45:349 0xF0D1B80000106F3D error APSDE-DATA.confirm: 0xD0 on task,
12:24:51:229 Current channel 11,
12:24:51:239 Device TTL 1957 s flags: 0x7,
12:24:56:139 0xF0D1B800000FF2FC error APSDE-DATA.confirm: 0xD0 on task,
12:24:56:252 dev /dev/ttyAMA0,
12:24:56:253 GW firmware version: 0x264a0700,
12:24:56:253 GW firmware version is up to date: 0x264a0700,
12:25:06:947 0x00212EFFFF032B30 error APSDE-DATA.confirm: 0xD0 on task,
12:25:17:842 0x00212EFFFF032B30 error APSDE-DATA.confirm: 0xD0 on task,
12:25:28:645 0x0017880106BAE90B error APSDE-DATA.confirm: 0xD0 on task,

Important information: before restart Conbee II worked fine with just a short 10cm USB cord extension (so ~10 cm from z-stick). When I saw those APSDE-DATA erros I've event replaced it with a 2m USB cord extension. But it doesn't change anything.

Please help, before my girlfriend will kill me for not able to switch off/on lights.

**Comment by manup** [comment]
Hey @tnowak since communication was through mail. Do you have any updates on the new setup, is it still up?

**Comment by tnowak** [comment]
Finally I had to reset all lights. It was somehow easy to reset Aqara and Osram bulbs, but to reset Philips bulbs I had to buy Philips Dimmer Switch. Dimmer has already come, I've reset last three bulbs and now all is online again and no problems since. I'm still worried because I don't know what has happened. I suspect Conbee and Aeontec z-stick might have changed their system names during reboot, but I'm not sure if this might brake things so hard. I've changed naming scheme for calling Conbee II to /dev/seria/by-id anyway. The other issue that worries me is that Phoscon/deCONZ does not report setup changes correctly (hidden page list). I would like to thank manup for helping me deal with this and despite we didn't find solution finally, we've exchange many emails during 2 days and such support from manufacturer is exceptional.
